forc/src/ops/forc_abi_json.rs

```diff
@@ -8,6 +8,7 @@ use crate::{
     },
 };
 
+use futures::executor::block_on;
```

Comment:
we are already using tokio's executor (in the main.rs - it's abstracted in the macro `#[tokio::main]`) the last thing we should do is add another executor